packages/material-components-web/index.ts

```diff
@@ -107,13 +102,12 @@ export {
   select,
   selectionControl,
   slider,
-  switchControl,
   snackbar,
+  switchControl,
   tab,
   tabBar,
   tabIndicator,
   tabScroller,
   textField,
-  toolbar,
```

Comment:
looks like you opened a PR for toolbar...i think you mean to revert this.

Reply:
Done